### DISCUSSION TITLE
Version history

### DISCUSSION THREAD

**Discussion by jantorep** [Azure/ipam#235] [Ideas]
Hello Team

Is it possible to get a version history that outlines the changes that get pushed out to the repo?

This time arround there was alot of changes. And it will take some time to read trough all the changes. Im not fluent in bicep and will never be. But i can read up on what's changed.

But a version history would speed up the process to see if there is anything i need. or should change. Or for new projects where this is used.

Thanks for an awsome repo

**Comment by DCMattyG** [comment]
Hi @jantorep, the changes that are currently being pushed will start to introduce Releases and history.

The simple answer is that nothing should really change, I've made it so the containers are backwards compatible with older releases, though there are also some newer deployment models to simplify the architecture.

Once the new workflow is working (hopefully very shortly) you'll see true Releases with proper version tagging and a complete change list.

In the meantime, is there something specific I can be of service with? I'm happy to answer any questions, or even to meet live (Teams) to go over your uses cases and the latest changes.

Just let me know!

**Comment by DCMattyG** [comment]
Just a quick update on this @jantorep, we're now properly versioning all releases starting with v3.0.0. Hopefully this help you moving forward and we'll make sure we put as many details in the Release Notes as possible. As always, please feel free to reach out should you need any additional details and we'll make sure you get the information you need.
